bootloader/src/arch/mipsel/boot.S

```diff
@@ -0,0 +1,20 @@
+.section .text.boot
+.globl _start
+
+_start:
+    # read cpu affinity, start core 0, halt the rest
```

Comment:
Waiting for multicore support

Reply:
not now

Reply:
will add multicore support, maybe later this month